x/evm/watcher/watcher.go

```diff
@@ -623,13 +802,23 @@ func (w *Watcher) lazyInitialization() {
 	w.jobDone = new(sync.WaitGroup)
 	w.jobDone.Add(1)
 }
+
 func (w *Watcher) Stop() {
 	if !w.Enabled() {
 		return
 	}
 	close(w.jobChan)
 	w.jobDone.Wait()
+	w.ACProcessorStop()
 }
+
+func (w *Watcher) ACProcessorStop() {
+	if GetEnableAsyncCommit() {
+		driveHeight := int64(w.height + 100) // use a greater height trigger last commit
```

Comment:
I saw `1<<63 - 1` is used by iavl/nodedb to identify the latest version.maybe we can use it here?

Reply:
> 1<<63 - 1

ok